csource/common.go

```diff
@@ -203,6 +225,8 @@ static void execute_command(const char* format, ...)
 
 int tunfd = -1;
 
+#define TUN_MTU 1500
```

Comment:
My understanding is that when a packet size is bigger than the MTU of the interface, it's split into fragments, and each of them is sent with proper headers. TUN/TAP MTU defaults to 1500, which is the maximum frame size for Ethernet packets, so one packet can't exceed this size.

I saw some mentions of so called jumbo frames with size up to 9000. As far as I understand, to enable jumbo frames we need to set MTU to 9000. I'm not sure whether it makes sense to deliberately increase the default MTU, since syzkaller can guess to do that by calling ioctls.